CRM/Admin/Page/ScheduleReminders.php

```diff
@@ -94,6 +94,7 @@ public function browse($action = NULL) {
         'ActionSchedule',
         $format['id']
       );
+      $format = array_merge(['class' => ''], $format);
```

Comment:
You really don't like the `+=` operator huh?

Reply:
@colemanw no I really don't - I thought about replying on the last one but then I tried to recall why - I know I've had unexpected results with it in the past but I can't recall any more than that

Reply:
I promise it's perfectly safe. I use it all the time :)